packages/paste-core/components/chat-log/stories/index.stories.tsx

```diff
@@ -28,31 +33,172 @@ export const OutboundChatMessage = (): React.ReactNode => {
   );
 };
 
+export const InboundMessageWithMeta = (): React.ReactNode => {
```

Comment:
ℹ️  You can import the following type from storybook and use it to type functional components used for stories. I recently started doing this and really like it because it provides more information specific to stories.

Something like this for the import, then the suggestion below 👇 
`import type {StoryFn} from '@storybook/react';`

```suggestion
export const InboundMessageWithMeta: StoryFn = () => {
```

Reply:
👍🏽